### DISCUSSION TITLE
Should we switch to log/slog

### DISCUSSION THREAD

**Discussion by fclairamb** [fclairamb/ftpserverlib#442] [Polls]
_This is mostly for @drakkan, but everyone else is welcome_

I created the [go-log](https://github.com/fclairamb/go-log) library because there was no standard library for structured logging in Go. This allowed us to create a common definition with some bindings to the most common structured logging libraries.

But since we now have one (log/slog), we should most probably switch to log/slog.

What do you think ?

**Comment by drakkan** [comment]
@fclairamb yes I think we should switch in the future, once we can drop support for Go 1.20

**Comment by fclairamb** [comment]
I'm thinking about keeping only compatibility with the two last releases. Most libraries do that. The rationale is that upgrading Go is a low-effort task.

**Comment by fclairamb** [comment]
This is done
